internal/factory/review.go

```
@@ -289,11 +289,37 @@ func reviewFindingBlocks(finding store.ReviewFinding) bool {
+		// rule.
+		evidence := strings.TrimSpace(finding.Evidence)
+		if strings.HasPrefix(evidence, "source=heuristic:") {
+			return false
+		}
+	}
+	return reviewFindingBlocks(finding)
+}
+
 // reviewFindingCounts separates concrete gating violations from visible
 // advisories for every supervision surface.
 func reviewFindingCounts(findings []store.ReviewFinding) (blocking, advisory int) {
```

golangci-lint (2.12.2)

[error] func reviewFindingCounts is unused

(unused)